SU-SOE/grunt-drupal-tasks
# D8CORE-580: Set HSTS

# READY FOR REVIEW

# Summary
- Just what it says on the tin

# Needed By (Date)
- ISO would like this by the end of the year, but if you don't have a release scheduled next week, it's fine to push it to the next release.

# Criticality
- How critical is this PR on a 1-10 scale? 6/10
- ISO wants it

# Steps to Test

1. Deploy using this branch
2. Check out the tag on dev/test
3. Clear Drupal and Varnish caches
4. `curl -I https://<url>` and check for the `Strict-Transport-Security: max-age=31536000` header (note: this header returns lowercase on some OSes and Title Case on others)

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- JIRA ticket: D8CORE-580
- Other PRs: yes

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
